pegasus/sites.v3/hourofcode.com/public/activity-guidelines.md

```diff
@@ -13,7 +13,7 @@ layout: wide
 
 # Lesson guidelines for the Hour of Code™ and Computer Science Education Week
 
-Code.org hosts a variety of Hour of Code™ activities, lessons, tutorials, and videos on the Code.org, Hour of Code, and CSEdWeek website(s). The current list is at [<%= resolve_url('code.org/learn') %>](<%= resolve_url('https://code.org/learn') %>).
+Code.org hosts a variety of Hour of Code™ activities, lessons, tutorials, and videos on the Code.org, Hour of Code, and CSEdWeek website(s). The current list is at [HourOfCode.com/learn](<%= resolve_url('/learn') %>).
```

Comment:
I think it's easier to read and we prefer it for things like #HourOfCode. So I think it helps to reinforce so people don't use #hourofcode (not like it matters _that_ much).